tests/dialects/test_sqlite.py

```diff
@@ -50,6 +50,13 @@ def test_ddl(self):
         )
 
     def test_sqlite(self):
+        self.validate_all(
```

Comment:
You can write tests for glob directly in fixtures/identity, because it's supported by the base parser, generator and glob expressions generally transpile back to themselves.

Reply:
jut put this test in identity.sql